<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="findButton">
       <property name="text">
        <string>&amp;Find</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="studentButton">
       <property name="text">
        <string>&amp;Student</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="courseButton">
       <property name="text">
        <string>&amp;Course</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="registerButton">
       <property name="text">
        <string>&amp;Registeration</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="gradeButton">
       <property name="text">
        <string>&amp;Grade</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="tableViewButton">
       <property name="text">
        <string>&amp;Table View</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>R&amp;eset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findButton</tabstop>
  <tabstop>studentButton</tabstop>
  <tabstop>courseButton</tabstop>
  <tabstop>registerButton</tabstop>
  <tabstop>tableViewButton</tabstop>
  <tabstop>gradeButton</tabstop>
  <tabstop>resetButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
